src/flow-util.c

```diff
@@ -201,6 +201,12 @@ void FlowInit(Flow *f, const Packet *p)
 
     f->protomap = FlowGetProtoMapping(f->proto);
 
+    if (f->macset != NULL) {
```

Comment:
In the `init` path, when will `f->macset` be non-NULL?

Reply:
Please correct me if I'm wrong: I just wanted to make sure that in case `FlowInit()` is called on a recycled flow, there is no memory leak my overwriting `f->macset` with a newly allocated instance. If this is not neessary here, I'd be happy to simplify the code.

Reply:
Ok ... This case happens alot (that is., `f->macset != NULL`.

Can memory be leaked by resetting a macset that has been extended with an add'l `SCCalloc`?

Reply:
I can't see where. A buffer, once created, is 
- never allocated again (only when it is `NULL` initially)
- never touched during the `MacSetReset()`
- always freed on `MacSetFree()` if it is not `NULL`.